server/api/calendar.go

```diff
@@ -27,7 +27,19 @@ func (api *api) CreateCalendar(calendar *remote.Calendar) (*remote.Calendar, err
 	return client.CreateCalendar(calendar)
 }
 
-func (api *api) CreateEvent(calendarEvent *remote.Event) (*remote.Event, error) {
+func (api *api) CreateEvent(calendarEvent *remote.Event, IDs []string) (*remote.Event, error) {
```

Comment:
```suggestion
func (api *api) CreateEvent(calendarEvent *remote.Event, userIDs []string) (*remote.Event, error) {
```

Reply:
I'd recommend going as far as establishing a convention of a `MattermostUserID` vs `RemoteUserID` and complying with it throughout. Names should be intention revealing, but most importantly consistent - same name should mean the same thing. Practical exceptions are often made for field names, but for variables like this I don't think it's a good excuse. Code should read like a book - clear, but not much redundancy :)

On that note why not s/calendarEvent/event/? What is `calendar` referring to here that is not already implied by `event`?

Reply:
I prefer `MattermostUserID` for this name because there are several `IDs` (calendar, MM UserId, Outlook User ID)

Also agree with `s/calendarEvent/event/` as `event` does imply a calendar event.